src/vm-repair/azext_vm_repair/_params.py

```diff
@@ -0,0 +1,32 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.arguments import CLIArgumentType
+
+from azure.cli.command_modules.vm._actions import _resource_not_exists
+from azure.cli.core.commands.parameters import get_resource_name_completion_list
+
+def load_arguments(self, _):
+
+    # REUSABLE ARGUMENT DEFINITIONS
+    name_arg_type = CLIArgumentType(options_list=['--name', '-n'], metavar='NAME')
+    existing_vm_name = CLIArgumentType(overrides=name_arg_type,
+                                       configured_default='vm',
+                                       help="The name of the Virtual Machine. You can configure the default using `az configure --defaults vm=<name>`",
+                                       completer=get_resource_name_completion_list('Microsoft.Compute/virtualMachines'), id_part='name')
+    existing_disk_name = CLIArgumentType(overrides=name_arg_type, help='The name of the managed disk', completer=get_resource_name_completion_list('Microsoft.Compute/disks'), id_part='name')
+
+    with self.argument_context('vm repair') as c:
+        c.argument('vm_name', existing_vm_name)
+
+    with self.argument_context('vm repair swap-disk') as c:
+        c.argument('rescue_username', help='Optional Admin username for rescue VM. Prompt will pop up if not given.')
```

Comment:
maybe just name these username and password and add these under an arg group. These arguments would be grouped under the arg group in help text. This is not necessary, but it makes it a bit clearer that these args are related.

```suggestion
        c.argument('username', arg_group="Rescue Authentication", 
        help='Admin username for rescue VM..')
        c.argument('password', arg_group="Rescue Authentication", 
        help='Admin password for the rescue VM.')    
```

Wrapped lines so you see the more concise help.